src/Components/AdvancedSearch/AdvancedSearch.stories.tsx

```diff
@@ -12,13 +16,29 @@ export default {
     decorators: [getDefaultStoryDecorator<IAdvancedSearchProps>(wrapperStyle)]
```

Comment:
can we add an interactive test that adds a property of each type so that when we get back Chromatic it will snapshot the different visual styles of each editor?